app/models/occupation_standard.rb

```diff
@@ -49,6 +51,36 @@ def clone_as_unregistered!(creator_id:, organization_id:)
     end
   end
 
+  def should_generate_attachment?(kind)
+    # Occupation standard updated_at timestamp gets set milliseconds after
+    # attachment created_at timestamp.
+    !send(kind).attached? || send(kind).created_at < updated_at - 1.second
+  end
+
+  def to_csv
+    CSV.generate do |csv|
+      csv << %w(rapids_code organization_title occupation_standard_title type work_process_title work_process_description work_process_hours work_process_sort skill skill_sort)
+      occupation_standard_work_processes.includes(:work_process, :occupation_standard_skills, :skills).each do |oswp|
```

Comment:
If you think we'll end up using this `includes` in a few places (per discussion of how the api should work on the call today), should we make this a scope so we dont have to write the whole thing out each time?